### PR TITLE
AI-52 Create a Top Level Run Script (Integration Continued)

### PR DESCRIPTION
**Pull request by msettro**

### CODE REVIEW

File: conf/iopstest.conf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#
+# file: iopstest.conf
+# description: maps test objects high level concepts to provision a test
+#
+
+cassandra test1  machine1 WriteIntensiveLoad
```
Comment (kliu17) [inline conf/iopstest.conf]: clarification: 
1. does each test is just loading the test configuration or will it fire up the test automatically after loading?
2. do we see parameterize the machine name, the number of tests, the VM instance types (e.g. new machine types), and the work loads in the future

File: iopstest [hunk 2]
```diff
@@ -3,129 +3,71 @@
...
 
-THIS=$1
+function iopstest_command() {
 
-function iopstest() {
-
-   function debug() {
-     echo "debug: $*"
+   function _usage() {
+     echo "usage: iopstest <action> <database> <test>,<test>,...<testN>"
```
Comment (kliu17) [inline iopstest]: is this usage still our target?

File: specs/machine/machine4.env.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export TF_VAR_machine_name="i3.4xlarge NVMe"
+export TF_VAR_machine_instance_type="i3.4xlarge"
+export TF_VAR_machine_root_volume_size=50
+export TF_VAR_machine_root_volume_type="gp2"
+export TF_VAR_machine_root_delete_on_termination="true"
+
+export TF_VAR_machine_ebs_device_name="/dev/xvdh"
```
Comment (kliu17) [inline specs/machine/machine4.env.sh]: this needs to be revised for the NVMe disk

File: terraform-common/common_machine.tf [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  connection {
+    user        = "${var.common_username}"
+    private_key = "${var.common_key_path}"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "sudo mkfs -t xfs /dev/xvdh"
```
Comment (kliu17) [inline terraform-common/common_machine.tf]: how about the i3.4xlarge disk type?

File: specs/machine/machine4.env.sh (hunk 3 above)
Comment (msettro) [reply specs/machine/machine4.env.sh]: Yes, coming soon...

File: terraform-common/common_machine.tf (hunk 4 above)
Comment (msettro) [reply terraform-common/common_machine.tf]: Also coming soon...

**Pull request merged**